admin/release.py

```diff
@@ -372,27 +372,38 @@ def update_repo(rpm_directory, target_bucket, target_key, source_repo,
     :param distro_version: The distro_version of the distribution to upload
         packages for.
     """
-    rpm_directory.createDirectory()
+    package_directory.createDirectory()
+
+    from .packaging import Distribution
```

Comment:
I guess / hope that when https://clusterhq.atlassian.net/browse/FLOC-1223 is merged this can be moved to the top of the file.